src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -28,7 +28,7 @@ import {
 } from '@chakra-ui/react';
 
 const ContactUs = () => {
-  const business_ID = 0;
+  const business_ID = 1;
```

Comment:
make sure its business id 0 because business id 0 sends it to FPH

Reply:
unless you think me and josh are wrong then please let us know why this is correct